flake8_quotes/__init__.py

```diff
@@ -30,13 +30,15 @@ def __init__(self, tree, filename='(none)', builtins=None):
 
     @classmethod
     def add_options(cls, parser):
-        parser.add_option('--quotes', default='\'', action='store',
+        parser.add_option('--quotes', default='\'', action='store', dest='inline_quotes',
```

Comment:
I think the combination of 9877c12585759c4d0b2269a75c8797f0ff83f76c and d6dad0be845ecb0950109399474e2c629d2fa532 means both `quotes` and `inline-quotes` are valid options for flake8 and should work as expected?